lib/backend/k8s/k8s_test.go

```diff
@@ -589,13 +578,6 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 			},
 		}
 
-		// Make sure we clean up after ourselves.  We allow this to fail because
-		// part of our explicit testing below is to delete the resource.
-		defer func() {
-			gnpClient.Delete(ctx, kvp1a.Key, "")
```

Comment:
where is the equivalent code for this (in `AfterEach`?)?

Reply:
This is part of the `c.Clean` call which cleans up all Calico resources.

Reply:
ah yes! right right